requires-dev.txt

```diff
@@ -1,14 +1,15 @@
 dash_flow_example==0.0.5
 dash-dangerously-set-inner-html
-isort==4.3.21
-mock==4.0.1;python_version>="3.0"
+isort==4.3.21;python_version<"3.7"
+mock==4.0.3;python_version>="3.0"
 mock==3.0.5;python_version=="2.7"
-flake8==3.7.9
-PyYAML==5.3
-pylint==1.9.4;python_version<"3.7"
-pylint==2.3.1;python_version=="3.7"
-astroid==2.2.5;python_version=="3.7"
+flake8==3.8.4
+PyYAML==5.3.1
+pylint==1.9.5;python_version<"3.7"
+pylint==2.6.0;python_version>="3.7"
+astroid==2.4.2;python_version>="3.7"
 black==19.10b0;python_version>="3.0"
-virtualenv==20.0.10;python_version=="2.7"
-fire==0.2.1
-coloredlogs==14.0
+virtualenv==20.2.2;python_version=="2.7"
+fire==0.3.1
+coloredlogs==15.0
+flask-talisman==0.7.0
```

Comment:
Moved `flask-talisman` here from `requires-testing` because this is used in our tests themselves, not in `dash.testing`.